gtsam/gtsam.i

```diff
@@ -2294,10 +2358,14 @@ class Values {
                  gtsam::Cal3_S2,
                  gtsam::Cal3DS2,
                  gtsam::Cal3Bundler,
-                 gtsam::EssentialMatrix,
+                 gtsam::Cal3Fisheye,
+                 gtsam::Cal3Unified,
+		 gtsam::EssentialMatrix,
                  gtsam::PinholeCameraCal3_S2,
-                 gtsam::PinholeCamera<gtsam::Cal3Bundler>,
-                 gtsam::imuBias::ConstantBias,
+                 gtsam::PinholeCameraCal3Bundler,
+                 gtsam::PinholeCameraCal3Fisheye,
+                 gtsam::PinholeCameraCal3Unified,
+		 gtsam::imuBias::ConstantBias,
```

Comment:
Maybe make the formatting consistent? I was wondering what was going on here for a second.